lib/Command/ConfigReport.php

```diff
@@ -30,8 +30,25 @@
  */
 class ConfigReport extends Command {
 
-	public function __construct() {
+	/**
+	 * @var ReportDataCollector
+	 */
+	private $reportDataCollector;
 
+	public function __construct() {
```

Comment:
more lazy as in: "instantiate the ReportDataCollector in the `run()` method, not the constructor"